openstack/workflow/v2/executions/requests.go

```diff
@@ -64,3 +67,55 @@ func Delete(client *gophercloud.ServiceClient, id string) (r DeleteResult) {
 	_, r.Err = client.Delete(deleteURL(client, id), nil)
 	return
 }
+
+// ListOptsBuilder allows extension to add additional parameters to the List request.
+type ListOptsBuilder interface {
+	ToExecutionListQuery() (string, error)
+}
+
+// ListOpts filters the result returned by the List() function.
+type ListOpts struct {
```

Comment:
Same as the other PR. It looks like there are some missing fields as well as possibly renaming some fields:

https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/execution.py#L321-L327